### ISSUE TITLE
update latest docker image to latest v0.15

### ISSUE THREAD

**Issue by aavileli** [timberio/vector#8418]
HI Devs

Please update the latest docker image **timberio/vector:latest-alpine** to 0.15 as we are having memory leaks which has been fixed based on our testing. The lastest is still bringing in 0.14

![image](https://user-images.githubusercontent.com/9123144/126586842-c6b035d7-21f8-433d-9d71-dbd57b4c62ad.png)

docker run --rm timberio/vector:latest-alpine --version                                                                                                                       
vector 0.14.1 (x86_64-unknown-linux-musl f9212bc 2021-07-20)

I have changed our tag temporarily to **timberio/vector:0.15.0-alpine** 

our config 
```toml
[sinks.prometheus_sink]
address="0.0.0.0:9090"
inputs=["internal_metrics"]
type="prometheus"
[sources.source_s3]
compression="gzip"
region="ap-southeast-2"
type="aws_s3"
sqs.delete_message=true
sqs.poll_secs=1
sqs.queue_url="https://sqs.ap-southeast-2.amazonaws.com/xxxxxxx/xxxx-vector-aggregator-s3-ingestion-buffer"
[transforms.extract_message]
inputs=["source_s3"]
type="remap"
source='.=parse_json!(.message)'
[transforms.route]
inputs=["extract_message"]
type="route"
route.es='exists(.index)'
route.cloudwatch='exists(.cloudwatch_group)'
[sinks.out_cloudwatch]
inputs=["route.cloudwatch"]
type="aws_cloudwatch_logs"
compression="gzip"
create_missing_group=true
create_missing_stream=true
group_name="/{{Type}}/{{Name}}/{{Tier}}/{{cloudwatch_group}}-{{index}}"
region="ap-southeast-2"
stream_name="{{host}}"
encoding.codec="json"
#Disablinghealthchecksherebecauseitwillendupperformingalogs:DescribeLogGroupsonanemptylogstream.
healthcheck.enabled=false
[sinks.out_elasticsearch]
inputs=["route.es"]
type="elasticsearch"
endpoint="https://xxxxxxx:443"
index="{{index}}-%Y.%m.%d"
healthcheck.enabled=true 
```

Thanks

**Comment by jszwedko** [comment]
Hi @aavileli !

Apologies, we released 0.14.1 yesterday with a fix backported neglecting that it would update the `latest` docker tags. These have been fixed to point to 0.15.0.

Just as a note, I'd recommend not using the `latest` tag in deployments as we do occasionally make breaking changes between minor versions of Vector (e.g. 0.14 -> 0.15) while we are pre-1.0. We aim to keep these to a minimum, but they do exist.

I'll close this out, but let us know if it still isn't working for you!
